.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+ ## Pull Request Checklist
+
+* [ ] Have you followed the steps in our [Contributing](../../blob/master/docs/CONTRIBUTING.md#steps-to-contribute) document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
+* [ ] Have you successfully ran tests with your changes locally?
+* [ ] Have you written new tests for your changes, if necessary?
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [ ] Have you made improvements to the documentation, if necessary? ([docs/](../../blob/master/docs/) folder and [README.md](../../blob/master/README.md))
+* [ ] Have you link the correct [GitHub issue](../../issues), if contains?
```

Comment:
Shouldn't it be 'linked', instead of 'link'
And, 'if there is any', instead of 'if contains'

Reply:
done.